playbooks/utils/security_theater.yml

```diff
@@ -0,0 +1,95 @@
+---
+#  This playbook will install OIT recommended security tools Rapid7 and Crowdstrike
+#
+- name: install OIT Security Tools on a host
+  hosts: staging:qa:production
+  remote_user: pulsys
+  serial: "{{ concurrent_vms | default('5') }}"
+  become: true
+  vars_files:
+    - ../../group_vars/crowdstrike/vault.yml
+    - ../../group_vars/crowdstrike/vars.yml
+    - ../../group_vars/all/vars.yml
+    - ../../group_vars/all/vault.yml
+
+  tasks:
+  - name: Check if the service is running
+    ansible.builtin.service_facts:
+
+  - name: Print service facts
+    ansible.builtin.debug:
+      var: ansible_facts.services
+
+  - name: Print falcon-sensor.service name
+    ansible.builtin.debug:
+      var: ansible_facts.services['falcon-sensor.service'].name
+
+  - name: Print falcon-sensor status name
+    ansible.builtin.debug:
+      var: ansible_facts.services['falcon-sensor.service'].status
```

Comment:
We can remove these tasks, or at least comment them out. We only needed them to see what the variable we needed looked like.